examples/sdk/reactNative/README.md

```diff
@@ -19,3 +24,38 @@ android-sourcemap.sh script.
 
 The script will prepare a release APK version of your React Native application with the sourcemap and Hermes support.
 The APK can be found in the ./android/app/build/outputs/apk/release/ directory.
+
+On iOS: Backtrace simplify the flow needed to upload source maps by instructing hermesc how to generate and prepare
+source maps for your application. Without this additional step, react-native will not instruct your application to
+support Backtrace source maps.
+
+In order to prepare your application for source maps and automatically upload them to Backtrace, modify your "Build
+Phase" with the following code:
+
+```bash
+set -e
+
+# destination source map directory
+SOURCE_MAP_DIR="$(pwd)/../build"
+mkdir -p $SOURCE_MAP_DIR
+
+export SOURCEMAP_FILE="$SOURCE_MAP_DIR/main.js.map";
+WITH_ENVIRONMENT="../node_modules/react-native/scripts/xcode/with-environment.sh"
+REACT_NATIVE_XCODE="../node_modules/react-native/scripts/react-native-xcode.sh"
+
+# use hermesc script provided by Backtrace to populate source maps
+# if you dont use hermes support, please skip this step.
+export HERMES_CLI_PATH="$(pwd)/../ios-hermesc.sh"
+
+/bin/sh -c "$WITH_ENVIRONMENT $REACT_NATIVE_XCODE"
+
+# copy javascript build output to the build directory
+cp "$CONFIGURATION_BUILD_DIR/main.jsbundle" $SOURCE_MAP_DIR
+
+# process source map with javascript code
+backtrace-js run --config "$(pwd)/../.backtracejsrc" --path "$SOURCE_MAP_DIR/main.jsbundle"
```

Comment:
maybe we should use `npx` here? 

```
npx --yes backtrace-js run --config "$(pwd)/../.backtracejsrc" --path "$SOURCE_MAP_DIR/main.jsbundle"
```

(the `--yes` switch will install `backtrace-js` if it is not available).

Reply:
Good idea!